moor/lib/src/runtime/query_builder/expressions/exists.dart

```diff
@@ -0,0 +1,35 @@
+part of '../query_builder.dart';
+
+class _ExistsExpression<T> extends Expression<bool> {
+  final BaseSelectStatement _select;
+  final bool _not;
+
+  @override
+  Precedence get precedence => Precedence.comparisonEq;
+
+  _ExistsExpression(this._select, this._not);
+
+  @override
+  void writeInto(GenerationContext context) {
+    if (_not) {
+      context.buffer.write('NOT ');
+    }
+    context.buffer.write('EXISTS ');
+
+    context.buffer.write('(');
+
+    _select.writeInto(context);
+
+    context.buffer.write(')');
```

Comment:
I don't think we need empty lines between every line here, one after the `EXISTS` should be good enough


```suggestion
    context.buffer.write('(');
    _select.writeInto(context);
    context.buffer.write(')');
```